.github/workflows/update-build-files.yml

```diff
@@ -33,22 +33,25 @@ jobs:
         ref: english
         fetch-depth: 0
 
-    - name: Download WordPress build files from the repository branch 'trunk'
-      id: download-files
+    - name: Checkout WordPress/wordpress-develop
+      uses: actions/checkout@v4
+      with:
+        repository: WordPress/wordpress-develop
+        fetch-depth: 1
+        path: wordpress-develop
+
+    - name: Copy WordPress build files
       run: |
         mkdir downloads
-        curl -o downloads/readme.html https://raw.githubusercontent.com/WordPress/wordpress-develop/trunk/src/readme.html
-        curl -o downloads/wp-config-sample.php https://raw.githubusercontent.com/WordPress/wordpress-develop/trunk/wp-config-sample.php
+        cp --force wordpress-develop/src/readme.html downloads/readme.html
+        cp --force wordpress-develop/wp-config-sample.php downloads/wp-config-sample.php
 
     - name: Check if source files have changed
       id: compare-files
       run: |
         if ! cmp -s downloads/readme.html en_US/readme.html || ! cmp -s downloads/wp-config-sample.php en_US/wp-config-sample.php; then
           echo "The build files have been updated!"
           echo "files_changed=true" >> $GITHUB_OUTPUT
-        else
-          echo "The build files remain unchanged."
-          echo "files_changed=false" >> $GITHUB_OUTPUT
```

Comment:
This is unused code.

Reply:
This code is used.
https://github.com/wp-portugal/wp-portugal-i18n-build-files/actions/runs/8714352677/job/23904450109

![imagem](https://github.com/wp-portugal/wp-portugal-i18n-build-files/assets/7371591/3a8f1c44-0c50-4ece-96c4-d78f1d9d926d)

Reply:
I mean the output variable is never checked for "false", only for "true".